src/SonarScanner.Shim/PropertiesWriter.cs

```diff
@@ -284,6 +284,27 @@ private static bool IsAscii(char c)
             return c <= sbyte.MaxValue;
         }
 
+        internal /* for testing purposes */ string JoinAsSonarQubeMultiValueProperty(IEnumerable<string> paths)
```

Comment:
This behavior needed to be replicated for all multi-value properties so I extracted it out.

Reply:
Are all of the multi-valued properties paths? (wondering whether the same warning message makes sense for other properties)

Reply:
As far as I have seen they are.